cfme/tests/middleware/test_middleware_provider.py

```diff
@@ -43,7 +44,7 @@ def test_add_cancelled_validation():
 def test_password_mismatch_validation(soft_assert):
     """ Tests password mismatch check """
     prov = HawkularProvider()
-    cred = prov.Credential(
+    cred = Credential(
```

Comment:
diro with all those - this seems to be a unintended refactoring automation artifact